src/Documentation/Getting-Started-With-Orleans/toc.yml

```diff
@@ -4,10 +4,6 @@
   href: Developing-a-Grain.md
 - name: Developing a Client
   href: Developing-a-Client.md
-- name: Grain Persistence
-  href: Grain-Persistence.md
-- name: Observers
-  href: Observers.md
 - name: Running the Application
   href: Running-the-Application.md
 - name: Silos
```

Comment:
If you look at #3296, that's kind of how I laid out them, except I got rid of Clients and Silos as separate pages because I think they don't have enough content, and the concept is better explained within the bigger architectural picture of the remaining pages.